npm/test/filechecks.test.ts

```diff
@@ -41,7 +41,7 @@ test("updateFileCheckProgress updates the progress bars correctly", () => {
   mockConsignmentData({
     antivirusProcessed: 1,
     checksumProcessed: 2,
-    totalFiles: 1
+    totalFiles: 2
   })
   fileChecks.updateFileCheckProgress()
   jest.runOnlyPendingTimers()
```

Comment:
This test looks like it's failing on Jenkins, I think the expected data may also need changed

Reply:
Yep, my fault. I've updated the expected result.